utils/storage/storage.js

```diff
@@ -97,6 +83,25 @@ class Storage {
     return value;
   }
 
+  setCookieDomain(value) {
+    const Cookie = new CookieLocal({});
+    Cookie.set("rudder_cookies", true);
```

Comment:
Can we create a config file within utils and add an Enum which will contain key value pair for all custom string that we are hard coding.
Example: EnumCookieConfig : {
RudderCookie: "rudder_cookies",
CookieNotSet: "not set",
CustomDomain: "custom_cookie_domain"
TestRudder: "test_rudder"
}